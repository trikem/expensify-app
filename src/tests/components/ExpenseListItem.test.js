import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../components/ExpenseListItem';
import expenses from '../fixtures/expenses';

test('should render exp list', () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[2]} />);
    expect(wrapper).toMatchSnapshot();
});