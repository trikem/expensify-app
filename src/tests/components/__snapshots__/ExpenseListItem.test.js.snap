// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render exp list 1`] = `
<div>
  <Link
    replace={false}
    to="edit/3"
  >
    <h3>
      Credit Card
    </h3>
  </Link>
  <p>
    $50.00
    -
    1970 January 4th
  </p>
</div>
`;
